### DISCUSSION TITLE
What letter comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#518] [Galaxy Trivia] [answered]
A O E A O T F ?

**Comment by NobleOsinachi** [comment]
S

**Comment by peterbaccaro** [comment]
This is a sequence of letters that represent the last letters of the phonetic alphabet.
So the next letter in the sequence will be **L**.

Alph**a**, Brav**o**, Charli**e**, Delt**a**, Ech**o**, Foxtro**t**, Gol**f**, Hote**l** ...

**Comment by thuongtruong1009** [comment, marked as answer]
The given sequence is: A O E A O T F.

The next letter in the sequence is likely to be "S".

The sequence follows the pattern of the first letter of each word in the sentence, "A Once Easy Attempt to Follow". The next word in this sentence begins with the letter "S", so it is likely that the next letter in the sequence is "S".

Therefore, the complete sequence would be: A O E A O T F S.
